.github/workflows/build-all-arduino.yml

```
@@ -0,0 +1,121 @@
+      tag_name: ${{ steps.extract-tag.outputs.tag_name }}
+    steps:
+      - name: Extract tag name
+        id: extract-tag
+        run: |
+          tag_name="${GITHUB_REF#refs/tags/}"
+          echo "tag_name=${tag_name}" >> $GITHUB_OUTPUT
+          echo "Extracted tag: ${tag_name}"
+
+      - name: Check if release exists
+        id: check
+        uses: actions/github-script@v6
```

actionlint (1.7.7)

the runner of "actions/github-script@v6" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)